eslint.config.mjs

```
@@ -0,0 +1,80 @@
+import globals from "globals";
+import pluginJs from "@eslint/js";
+import tseslint from "typescript-eslint";
+import pluginReact from "eslint-plugin-react";
+import stylisticJs from '@stylistic/eslint-plugin-js';
+
+
+/** @type {import('eslint').Linter.Config[]} */
+export default [
+  {
```

ESLint

[error] Expected indentation of 4 spaces but found 2.

(@stylistic/js/indent)